negativ_groups/static/description/index.html

```diff
@@ -0,0 +1,45 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <p class="oe_mt32">
+
+            <h2>
+                <center>
+                    Provide Negativ groups for odoo
+                </center>
+            </h2>
+        </div>
+
+        <div class="oe_span12">
+            <p class="oe_mt32">
+                You can now use in the backend and frontend groups and
+                negative groups.</p>
+
+            <p class="oe_mt32"> Example:
+                <code>groups="myaddon.group"</code> and now the new
+                <code>groups="not myaddon.group"</code>.
+            </p>
+        </div>
+
+    </div>
+</section>
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+        <h2 class="oe_slogan"><a href="http://www.bloopark.de/">bloopark
+            systems GmbH &amp; Co. KG</a></h2>
+
+        <div class="oe_span12">
+            <p class="oe_mt32">
+                We are an internet agency and Odoo partners based in Magdeburg,
+                Germany with over 10 years of experience
+                in e-commerce, responsive design and development. We consider
+                our selves and A-team, small and
+                efficient. With remote developers all across de world, we pride
+                ourselves of our wide perspective and
+                approach in every challenge, which leads to the best
+                customer-oriented solutions.
+            </p>
```

Comment:
I'm not the author company description should be in an OCA module.
I also dislike having a description that repeats the text in the README.